src/mapping/lending-pool/lending-pool.ts

```diff
@@ -69,7 +69,7 @@ export function handleDeposit(event: Deposit): void {
 export function handleWithdraw(event: Withdraw): void {
   let toUser = getOrInitUser(event.params.to);
   let poolReserve = getOrInitReserve(event.params.reserve, event);
-  let userReserve = getOrInitUserReserve(event.params.user, event.params.reserve, event);
+  let userReserve = getOrInitUserReserve(event.params.to, event.params.reserve, event);
```

Comment:
The redeemUnderlying fields should be adjusted for this as well. The `user` and `onBehalfO`f are both set to the `to` address now